.golangci.reference.yml

```diff
@@ -1543,145 +1544,176 @@ linters-settings:
       - name: argument-limit
         severity: warning
         disabled: false
+        exclude: [""]
         arguments: [ 4 ]
       # https://github.com/mgechev/revive/blob/master/RULES_DESCRIPTIONS.md#atomic
       - name: atomic
         severity: warning
+        exclude: [""]
         disabled: false
       # https://github.com/mgechev/revive/blob/master/RULES_DESCRIPTIONS.md#banned-characters
       - name: banned-characters
         severity: warning
         disabled: false
+        exclude: [""]
```

Comment:
inconsistent with `arguments` format, e.g. 

```
[ "Ω","Σ","σ", "7" ]
[ 4 ]
```

`[ "" ]` expected

Reply:
yes because there is no example inside the revive doc, and it's complex to suggest right values for this option.